include/libdnf-cli/output/transaction_table.hpp

```diff
@@ -300,7 +300,11 @@ bool print_transaction_table(Transaction & transaction) {
         scols_line_set_data(ln, COL_NAME, pkg.get_name().c_str());
         scols_line_set_data(ln, COL_ARCH, pkg.get_arch().c_str());
         scols_line_set_data(ln, COL_EVR, pkg.get_evr().c_str());
-        scols_line_set_data(ln, COL_REPO, pkg.get_repo_id().c_str());
+        auto tspkg_repo = pkg.get_repo_id();
+        if (tspkg.get_action() == libdnf::transaction::TransactionItemAction::REMOVE) {
+            tspkg_repo = "<TODO: from_repo>";
+        }
```

Comment:
The value tspkg_repo is initialized with one value and then ovewritten when condition is met. What about to set the value directly to the column?
```
       if (tspkg.get_action() == libdnf::transaction::TransactionItemAction::REMOVE) {
            scols_line_set_data(ln, COL_REPO, "<TODO: from_repo>");
        } else {
            scols_line_set_data(ln, COL_REPO, pkg.get_repo_id().c_str());
        }
```

Reply:
IMO what you've suggested is prone to errors, because there are 2 lines setting COL_REPO data and in case we change only one of them in the future we end up with an inconsistency. But I don't have any strong opinion on this so I'm ok witch changing it.

Reply:
Changed.